contents/incentivization.rst

```diff
@@ -0,0 +1,189 @@
+
+.. image:: img/cheque.svg
+   :alt: flow diagram depicting how cheques are triggered 
+   :width: 400
+
+Interaction with the blockchain
+-------------------------------
+A cheque is sent as part of its own lower layer transport protocol (currently on top of `devp2p`, with the planned transition of the whole of Swarm to `libp2p`). The receiving peer handles the cheque and tries to cash it by issuing a transaction on the blockchain. 
+
+Every node has to deploy its own instance of the Swarm smart contract, also often referred to as *chequebook*.  So when receiving a cheque, the beneficiary issues a transaction *on the contract of the issuer of the cheque* as a cashing in request. As the beneficiary is initiating the transaction, it is also the beneficiary who is paying for the transaction. Swarm per default has a check to make sure it financially makes sense to do this: currently the transaction is only started if the payout of the cheque is twice as big as the (estimated) transaction costs. 
+
+The smart contract, after doing the appropriate sanity checks (checking the validity and the funds of the issuer), will transfer the funds from the issuer's contract to the beneficiary contract. If the cheque bounced due to insufficient funds in the issuer, the peer gets disconnected. Again, it is important to understand that a peer is able to freeride up to this threshold. The issuer cannot issue a cheque if there are insufficient funds; normal operation involves a smart contract call to check for funds. If a malicious node sends a bogus, malformed or altered cheque, the cheque will bounce and thus can be identified as fraud attempt, resulting in a disconnect.
+
+Refer to the documentation for details and specification of how the protocol handles subsequent cheques and how this evolves over time.
+
+Starting a SWAP enabled Swarm node thus requires a node to have funds. If no contract address is provided, the node will automatically create one, and a transaction is attempted when the node starts. The user can specify the amount to fund (in WEI) via command line parameter at boot (`swap-deposit-amount`). If the transaction failed, the node does not boot. Afterwards, the node remembers the contract address and uses it for future blockchain interaction.
+
+Currently, the node tries to cash in a cheque right away when it receives it via the protocol. In the future, Swarm might consider making this configurable by the user.
+
+To prevent fraudulent creation of contract addresses for the sake of stealing funds via the protocol, incentivized nodes need to contact an audited factory contract address when creating the new chequebook. For every blockchain, one factory should be deployed. For Ethereum networks, Swarm will provide the network address (and the deployment of the factory). The factory address needs to be configured via command line parameter for incentivized nodes (`swap-chequebook-factory`).
```

Comment:
In the last sentence you probably either mean "In other networks the factory address needs..." or "The factory address can be...".